api/lib/src/models/tool.dart

```diff
@@ -80,6 +80,7 @@ sealed class Tool with _$Tool {
     @Default('') String displayIcon,
     @Default(LabelMode.text) LabelMode mode,
     @Default(true) bool zoomDependent,
+    @Default(true) bool straightLineEnabled,
```

Comment:
please remove this here, label doesn't need it

Reply:
done